vector/src/main/java/im/vector/app/core/extensions/ConstraintLayout.kt

```diff
@@ -26,3 +30,16 @@ fun ConstraintLayout.updateConstraintSet(block: (ConstraintSet) -> Unit) {
         it.applyTo(this)
     }
 }
+
+fun ConstraintLayout.realignPercentagesToParent() {
```

Comment:
great point, will do! the tl;dr is that constraint layouts within scrollviews use the total scrolling height rather than the viewport, which means percentages become changes size depending on how much they scroll

Reply:
https://github.com/vector-im/element-android/pull/5648/commits/c45c421e48da62659f217e625ca69b6a65fd0adb